header.h

```diff
@@ -0,0 +1,141 @@
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+class BankAccount
+{
+private:
+
+
+    string Owner_name;
+    string Replenish_balance;
+    string Withdraw_cash;
+
+public:
+
+    int Balance;
+    int ID;
+    double amoumt;
+    int amount;
+    int Getid()
```

Comment:
Name of methods in class must be used like varriables
Like this ` int get_id()`